### PR TITLE
Species with no offensive Ground Troops will no longer be able to bui…

### PR DESCRIPTION
**Pull request by Deepsloth**
…ld ships with Ground Troop pods.

In response to: http://www.freeorion.org/forum/viewtopic.php?f=28&t=9762

### CODE REVIEW

File: default/ship_parts.txt [hunk 1]
```diff
@@ -8,7 +8,10 @@ Part
     buildcost = 3 * [[FLEET_UPKEEP_MULTIPLICATOR]]
     buildtime = 1
     tags = "COMFORTABLE"
-    location = OwnedBy empire = Source.Owner
+    location = And [
+        OwnedBy empire = Source.Owner
+        Not HasTag name = "AI_TAG_NO_ATTACKTROOPS"
```
Comment (geoffthemedio) [inline default/ship_parts.txt]: not really an "AI" tag then, is it...?

**Comment by Deepsloth** [comment]
Should i rename the tag to "TAG_NO_ATTACKTROOPS"?

**Comment by geoffthemedio** [comment]
Seems reasonable.

**Comment by MatGB** [comment]
The AI is using it though so it'd need changes in AI code, the other tag used is actually a macro so I assume that won't parse.

**Comment by Morlic-fo** [comment]
Tags used by the AI are required to start with AI_TAG unless someone feels like changing our parsing system - I certainly do not for a case like this that is not even related to the AI whatsoever.

If the name is a problem, then add another tag with the name you want to have and leave the AI tag intact.

**Comment by Deepsloth** [comment]
@geoffthemedio: Should i go with Morlics suggestion of creating a second tag for the same thing or should we use the initial implementation?

**Comment by geoffthemedio** [comment]
I suppose... there's not really an easier / better way to directly check a design's properties with a particular species in a location condition, as far as I know.

**Comment by Deepsloth** [comment]
@geoffthemedio: Your answer is ... a little unclear. I was asking an either/or question...

**Comment by geoffthemedio** [comment]
Add another tag

**Pull request merged**